packages/terra-i18n/src/I18n.js

```diff
@@ -1,4 +1,6 @@
-import { injectIntl, intlShape, FormattedMessage } from 'react-intl';
+import {
+  injectIntl, intlShape, FormattedMessage, intlProvider,
```

Comment:
Does this work? The IntlProvider is a component and starts with a capital I.

Reply:
Well this is awkward, I didn't actually test because I couldn't get `npm link` to work. But you are correct, this is incorrect https://github.com/cerner/terra-core/pull/1945/commits/7e190b372fc66e5bd6f646d189a2e481908bcdee